src/components/Calander/CalanderComponent.js

```diff
@@ -0,0 +1,351 @@
+import { React, useState, useEffect } from 'react';
+import { addDays, subDays } from 'date-fns';
+import { ko } from 'date-fns/esm/locale';
+import DatePicker from 'react-datepicker';
+import styled from 'styled-components';
+import 'react-datepicker/dist/react-datepicker.css';
+import theme from '../../styles/theme';
+
+const CalanderComponent = props => {
+  const [startDate, setStartDate] = useState(new Date());
```

Comment:
여기에 props에서 받아오는 값들은 모두 구조분해 할당 해주세요. 
이유는 eslint 에서의 warnning (선언형의 react에선 권고하지 않는 방식) 때문입니다.

```suggestion
const CalanderComponent = props => {
  const {makeInputToPriceFormat, getDateRange ... 등등 } = props;
  const [startDate, setStartDate] = useState(new Date());
```

이렇게 한줄에 적어주시면 더 깔끔하게 prop들을 관리 할 수 있습니다.

Reply:
수정완료했습니다!